app/js/app/directives/react_classes/PodBlock.js

```diff
@@ -0,0 +1,68 @@
+define(["react", "jquery"], function(React,$) {
+    return function(ContentEditor, Block, FigureBlock) {
+        return React.createClass({
+
+            getInitialState: function() {
+                return {
+                    value: this.props.doc.value,
+                }
+            },
+
+            onDocChange: function(c, oldDoc, newDoc) {
+                this.props.onChange(this, oldDoc, newDoc);
+            },
+
+            onValueChange: function(e) {
+                this.setState({
+                    value: e.target.value,
+                });
+
+                var oldDoc = this.props.doc;
+                var newDoc = $.extend({}, oldDoc);
+                newDoc.value = e.target.value;
+
+                this.onDocChange(this, oldDoc, newDoc);
+            },
+
+            onUrlChange: function(e) {
+                this.setState({
+                    url: e.target.value,
+                });
+
+                var oldDoc = this.props.doc;
+                var newDoc = $.extend({}, oldDoc);
+                newDoc.url = e.target.value;
+
+                this.onDocChange(this, oldDoc, newDoc);
+            },
+
+            onImageChange: function(c, oldVal, newVal) {
+                var oldDoc = this.props.doc;
+                var newDoc = $.extend({}, oldDoc);
+                newDoc.image = newVal;
+
+                this.onDocChange(this, oldDoc, newDoc);
+            },
+
+            render: function() {
+                return (
+                    <Block type="pod" blockTypeTitle="Pod" doc={this.props.doc} onChange={this.onDocChange}>
+                    <form>
+                    <div className="row">
```

Comment:
Are you missing a title here for physics pods?

Reply:
This was one of the 'I'll explain in a doc and keep it as we have now' - the page title is still there, it's in the metadata like other pages. I'd be happy to put it in as a field, it just would be duplicated